CHANGES.rst

```diff
@@ -8,7 +8,7 @@ Improvements:
  -
 
 Bugfix:
- -
+ - F-Droid: sync once before suspend the events stream when a minimum delay is defined between 2 sync requests.
```

Comment:
Not sure to understand. what is it fixing?

Reply:
My plan was to apply a delay after the sync and not before. 
I thought the check on `mPendingDelayedIntent` [here](https://github.com/matrix-org/matrix-android-sdk/pull/474/files#diff-b76f857b869ad5b3392d8806fa3f91b3L533) was not a the right place. But now I'm less sure...

Reply:
Let's have a short sync on that?